package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Render elements consistently. Style with best practices.",
   "author": "Jonathan Neal",
   "license": "CC0",
-  "main": "dist/sanitize.css",
```

Comment:
I'm wondering whether `main` should be kept. 
We discussed `style` because of some particular modules but it's not a "standard". 
I don't think having both would hurt.